Core/src/androidTest/java/com/paypal/android/core/HttpIT.kt

```diff
@@ -0,0 +1,37 @@
+package com.paypal.android.core
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@ExperimentalCoroutinesApi
+@RunWith(AndroidJUnit4::class)
+class HttpIT {
```

Comment:
Is `IT` the naming scheme we want to go with for integration test files? Seems like a good time to establish conventions on this. I'd vote for `HttpIntegrationTest`